azure-pipelines.yml

```diff
@@ -110,8 +122,8 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
     - task: PublishBuildArtifacts@1
-      displayName: Publish Release DMG
-      condition: eq(variables['B_BUILD_TYPE'], 'Release')
+      displayName: Publish Release DMG if on Big Sur agent
+      condition: and(eq(variables['B_BUILD_TYPE'], 'Release'), eq(variables['imageName'], 'macOS-11'))
       inputs:
         pathtoPublish: build/bundle
-        artifactName: Mac Release Disk Image and App $(imageName)
+        artifactName: Release DMG for macOS - no M1 support yet
```

Comment:
I don't think we should say this here. People should look into release notes. Artifact names may be used by scripts, so we should actually make them machine-readable long term.

Reply:
Resolved in 68283b90.